eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/responses/m2w/M2WebEwonEBDInstantValuesResponse.java

```diff
@@ -0,0 +1,69 @@
+package com.hms_networks.americas.sc.ignition.comm.responses.m2w;
+
+import com.opencsv.bean.CsvToBeanBuilder;
+import java.io.StringReader;
+import java.util.List;
+
+/**
+ * Object for an M2Web Ewon EBD instant values response.
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 2.0.0
+ * @version 1.0.0
+ */
+public class M2WebEwonEBDInstantValuesResponse {
+
+  /**
+   * The separator character used to separate columns/fields in the EBD instant values response.
+   *
+   * @since 1.0.0
+   */
+  private static final char EWON_EBD_INSTANTVALUES_SEPARATOR_CHAR = ';';
+
+  /**
+   * The list of instant values from the EBD instant values response.
+   *
+   * @since 1.0.0
+   */
+  private final List<M2WebEwonEBDInstantValue> instantValues;
+
+  /**
+   * Private constructor for an M2Web Ewon EBD instant values response.
+   *
+   * @param instantValues The list of instant values from the EBD instant values response.
+   * @since 1.0.0
+   */
+  private M2WebEwonEBDInstantValuesResponse(List<M2WebEwonEBDInstantValue> instantValues) {
+    this.instantValues = instantValues;
+  }
+
+  /**
+   * Gets the list of instant values from the EBD tag list response.
+   *
+   * @return The list of instant values from the EBD tag list response.
+   * @since 1.0.0
+   */
+  public List<M2WebEwonEBDInstantValue> getInstantValues() {
+    return instantValues;
+  }
+
+  /**
+   * Gets an instance of {@link M2WebEwonEBDInstantValuesResponse} from a response string.
+   *
+   * @param responseString The response string to parse.
+   * @return An instance of {@link M2WebEwonEBDInstantValuesResponse} parsed from the response
+   *     string.
+   * @since 1.0.0
+   */
+  public static M2WebEwonEBDInstantValuesResponse getFromString(String responseString) {
+    // Parse CSV response into list of tag objects
+    List<M2WebEwonEBDInstantValue> instantValues =
+        new CsvToBeanBuilder<M2WebEwonEBDInstantValue>(new StringReader(responseString))
+            .withSeparator(',')
+            .withType(M2WebEwonEBDInstantValue.class)
+            .build()
+            .parse();
```

Comment:
try/catch IllegalStateException ? 
https://opencsv.sourceforge.net/apidocs/com/opencsv/bean/CsvToBeanBuilder.html#build--
Or add Throws to function defintion?

Reply:
Fix will be included for the incorrect separator shown on line 62 as well.

Reply:
Changed

Reply:
throws was only added in Javadocs, not to method signature.